src/pages/studyView/SummaryHeader.tsx

```diff
@@ -74,9 +101,18 @@ export default class SummaryHeader extends React.Component<ISummaryHeaderProps,
                     <button className="btn" onClick={() => null}>
                         <i className="fa fa-download" aria-hidden="true" title="Download clinical data for selected cases"></i>
                     </button>
+                    <GeneSelectionBox
+                        inputGeneQuery={this.props.geneQuery}
+                        callback={this.updateSelectedGenes}
+                        location={GeneBoxType.STUDY_VIEW_PAGE}
+                    />
+                    {/* <span style={{ margin: "0px 5px" }}><i className="fa fa-arrow-right fa-lg" aria-hidden="true"></i></span>
+                    <button disabled={this.isQueryButtonDisabled} className="btn" onClick={() => null}>
+                        Query
+                    </button> */}
                     <button
-                        className="btn btn-default btn-sm"
-                        onClick={event=>this.isCustomCaseBoxOpen = true}
+                        className="btn"
+                        onClick={()=>this.isCustomCaseBoxOpen = true}
```

Comment:
for react to not rerender this unnecessarily, the value of onClick should be a defined function, because otherwise the anonymous function gets redefined on every render and always shows up as new props. its  probably not worth the effort here bc its just a single button but just wanted to point it out